lib/geo.js

```diff
@@ -138,11 +138,18 @@ function GeoPoint(data) {
     };
   }
 
-  assert(Array.isArray(data) || typeof data === 'object' || typeof data === 'string', 'must provide valid geo-coordinates array [lat, lng] or object or a "lat, lng" string');
+  assert(
+    Array.isArray(data) ||
+    typeof data === 'object' ||
+    typeof data === 'string',
+    'must provide valid geo-coordinates array [lat, lng] or object or a ' +
+      '"lat, lng" string'
+  );
```

Comment:
You are already doing it in other places in this patch, see e.g. https://github.com/strongloop/loopback-datasource-juggler/pull/928/files#diff-b367a815d81f15531e0d0553ea8aaa5aR152